mods/d2k/rules/vehicles.yaml

```diff
@@ -442,7 +442,7 @@ deviator:
 		BuildPaletteOrder: 40
 		BuildDuration: 373
 		BuildDurationModifier: 40
-		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry, Aircraft\n \n  Atreides:      +Range\n  Harkonnen:  +Health\n  Ordos:          +Speed
+		Description: Main Battle Tank\n  Strong vs Tanks\n  Weak vs Infantry
```

Comment:
Ahh I didn't know about TibEd... till now. 
You are right, there are no different types armament

Faction | "Strength" | ROF | Speed
|:-----:|:-----:|:-----:|:-----:|
Atreides | 2100 | 50 | 448
Harkonnen | 2700 | 55 | 384
Ordos | 1800 | 45 | 512